src/lang/fr.js

```diff
@@ -319,43 +319,43 @@ var SCRAMBLE_NOOBST = [
 ];
 var SCRAMBLE_NOOBSS = ' de 90 degrés dans le sens des aiguilles d\'une montre, | de 90 degrés dans le sens inverse des aiguilles d\'une montre, | de 180 degrés, ';
 var STATS_CFM_RESET = 'Effacer tous les temps de cette session ?';
-var STATS_CFM_DELSS = 'delete session [%s]?';
+var STATS_CFM_DELSS = 'Effacer la session [%s]?';
 var STATS_CFM_DELMUL = 'Nombre de valeurs effacées de l\'index en cours ?';
 var STATS_CFM_DELETE = 'Effacer ce temps ?';
 var STATS_COMMENT = 'Commentaire';
 var STATS_DATE = 'Date';
 var STATS_CURROUND = 'Statistiques de la moyenne';
 var STATS_CURSESSION = 'Statistiques de la session en cours';
-var STATS_CURSPLIT = 'Phase %d of Current Session Statistics';
+var STATS_CURSPLIT = 'Statistiques de la phase %d de la session en cours';
 var STATS_EXPORTCSV = 'Export CSV';
-var STATS_SSMGR_TITLE = 'Session Manager';
-var STATS_SSMGR_NAME = 'Name';
-var STATS_SSMGR_DETAIL = 'Session Details';
-var STATS_SSMGR_OP = 'Operation';
-var STATS_ALERTMG = 'Merge all times in session [%f] to the end of session [%t]?';
-var STATS_PROMPTSPL = 'Number of latest times split from session [%s]?';
-var STATS_ALERTSPL = 'Should split or leave 1 time at least';
+var STATS_SSMGR_TITLE = 'Gestion des sessions';
+var STATS_SSMGR_NAME = 'Nom';
+var STATS_SSMGR_DETAIL = 'Détails de la session';
+var STATS_SSMGR_OP = 'Opération';
+var STATS_ALERTMG = 'Déplacer tous les temps de la session [%f] à la fin de la session [%f]?';
+var STATS_PROMPTSPL = 'Nombre de temps à diviser de la session [%s] (en partant de la fin)?';
+var STATS_ALERTSPL = 'Il faut laisser au moins un temps, ou fusionner la session avec une autre';

```

Comment:
This is set to 'Should split or leave 1 time at least' in the English version, which didn't make sense to me.

I translated it as 'Should merge or leave 1 time at least', but I didn't update the English version. Should it be 'merge' instead of 'split'? If so then I can update the English file. If not then I'll fix this one.

Reply:
Well, it means that the new session and the old session should both contain 1 times at least after splitting.